app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -134,6 +134,17 @@ def download_statistics
     end
   end
 
+  # Gets the 10 most recent past answers of a specific question
+  def past_answers
+    if past_answers_params[:question_id]
+      answers = @submission.past_answers_from_question(
+        past_answers_params[:question_id], past_answers_params[:answers_to_load])
+      render 'course/assessment/submission/submissions/_answers.json.jbuilder', locals: { answers: answers }
```

Comment:
I think you can try something more rails-y like `render json: 'past_answer'` then use a partial to render the json data instead?